R/NitrogenBudget.R

```diff
@@ -41,18 +41,18 @@ NitrogenBudget<-function(gdx,level="reg",dir=".",spamfiledirectory="",debug=FALS
     bg <- res[,,"bg"]
     seed_detail <- Seed(gdx,level=level,attributes = "nr")
     seed <- dimSums(Seed(gdx,level=level,attributes = "nr"),dim=3)
-    
+
     ag_recycling <-dimSums(readGDX(gdx,"ov18_res_ag_recycling",select=list(type="level"))[,,"nr"],dim=c(3.1,3.2))
     ag_recycling <- gdxAggregate(gdx = gdx,weight = 'ResidueBiomass',x = ag_recycling, to = level,absolute = TRUE,dir = dir, product_aggr = T,attributes = "nr",plantpart="ag")
-    
-    ash<- dimSums((readGDX(gdx,"ov18_res_ag_burn",select=list(type="level"))[,,kcr]*(1-readGDX(gdx,"f18_res_combust_eff")[,,kcr]))[,,"nr"],dim=3)
+
+    ash <- dimSums((readGDX(gdx,"ov_res_ag_burn","ov18_res_ag_burn",select=list(type="level"))[,,kcr]*(1-readGDX(gdx,"f18_res_combust_eff")[,,kcr]))[,,"nr"],dim=3)
```

Comment:
I believe it must be `readGDX(..., format="first_found")` (you want to get either ov_res_ag_burn or ov18_res_ag_burn), right?